backend/swift/swift.go

```diff
@@ -276,6 +283,7 @@ var retryErrorCodes = []int{
 	401, // Unauthorized (e.g. "Token has expired")
 	408, // Request Timeout
 	409, // Conflict - various states that could be resolved on a retry
+	413, // Upload exceeds quota
```

Comment:
Is this something we want to retry? If we retry it then surely the upload will still exceed the quota?